content/en/docs/languages/php/instrumentation.md

```diff
@@ -124,6 +229,41 @@ Sdk::builder()
     ->buildAndRegisterGlobal();
 ```
 
+Include this file at the top of your application file `index.php`:
+
+```php
+<?php
+use Psr\Http\Message\ResponseInterface as Response;
+use Psr\Http\Message\ServerRequestInterface as Request;
+use Psr\Log\LogLevel;
+use Slim\Factory\AppFactory;
+use Monolog\Logger;
+use Monolog\Handler\StreamHandler;
+
+require __DIR__ . '/vendor/autoload.php';
+
+require('dice.php');
+require('instrumentation.php');
+
+// ...
+```
+
+For debugging and local development purposes, the example exports telemetry to
+the console. After you have finished setting up manual instrumentation, you need
+to configure an appropriate exporter to
+[export the app's telemetry data](/docs/languages/php/exporters/) to one or more
+telemetry backends.
+
+The example also sets up the mandatory SDK default attribute `service.name`,
+which holds the logical name of the service, and the optional (but highly
+encouraged!) attribute `service.version`, which holds the version of the service
```

Comment:
I think I've left similar comments in https://github.com/open-telemetry/opentelemetry.io/pull/3834 — we should sweep the docs everywhere to fix the issues...